gene/etl/base.py

```diff
@@ -94,7 +93,9 @@ def _load_gene(self, gene, batch) -> None:
         :param BatchWriter batch: Object to write data to DynamoDB
         """
         try:
+            gene['match_type'] = MatchType.NO_MATCH
             assert Gene(**gene)
```

Comment:
what about something like

```
assert Gene(match_type=MatchType.NO_MATCH, **gene)
```

Reply:
^ Tried that and getting TypeError, will just leave with adding/deleting the field

Reply:
Oh wait I forgot I set this in NCBI, I'll try removing that and that should fix it